integration/java/nexmark/BEAM_ON_FLINK_ON_GCP.md

```diff
@@ -0,0 +1,282 @@
+<!--
```

Comment:
It would be great to have it in the sdk! I am in favor of it. The only question I ask myself is: Nexmark is more a benchmarking suite, not an API or framework. Is it not a problem to have a piece of software that is not used to build software in the sdk module?